classes/Kohana/Jam/Validator/Rule/Count.php

```diff
@@ -20,7 +20,9 @@ class Kohana_Jam_Validator_Rule_Count extends Jam_Validator_Rule {
 
 	public function validate(Jam_Validated $model, $attribute, $value)
 	{
-		$count = count($value);
+		// Since PHP 7.2 method count() parameter must be an array or an object that implements Countable
+		// https://www.php.net/manual/en/function.count.php#124263
+		$count = count((array) $value);
```

Comment:
Ooh, this one could have been a bad one if there were no tests. Usually, casting a random value of an unknown type to an array like that could also lead to problems, but my quick test here looks ok: https://3v4l.org/X6D7n

Reply:
I was not able to discover a better approach. Do you think this is ok or can you suggest any change?

Reply:
No, I had doubts, but that's why I've checked and it looks fine for its purposes.